ServiceRepositoryPattern/ServiceRepositoryPattern/Migrations/ServiceRepositoryPatternContextModelSnapshot.cs

```diff
@@ -0,0 +1,76 @@
+﻿// <auto-generated />

```

Comment:
This is autogenerated file do you need to push it?

Reply:
These are migrations. Are they not important?